src/frontend/src/constant/constant.ts

```diff
@@ -1,16 +1,52 @@
 export const OtherFactor = [
-  'Sensitive personal information',
-  'Regular or systematic sharing of personal information',
-  'Common or integrated program agreement',
-  'Vendor or third-party access to personal information',
-  'Cloud technology',
-  'Potential public interest in the initiative',
-  'Disclosure of personal information outside of Canada',
-  'High volumes of personal information',
-  'Data-linking',
-  'BC Services Card Onboarding',
-  'Artificial intelligence (AI) or machine learning',
-  'Partnership with non-ministry public bodies or other organizations',
+  {
+    label: 'Sensitive personal information',
+    value: 'hasSensitivePersonalInformation',
+  },
+  {
+    label: 'Regular or systematic sharing of personal information',
+    value: 'hasSharingOfPersonalInformation',
+  },
+  {
+    label: 'Common or integrated program agreement',
+    value: 'hasProgramAgreement',
+  },
+  {
+    label: 'Vendor or third-party access to personal information',
+    value: 'hasOthersAccessToPersonalInformation',
+  },
+  {
+    label: 'Cloud technology',
+    value: 'hasCloudTechnology',
```

Comment:
MinistryList is following a different pattern than what you have here. Was that intentional?

Reply:
yes, it is the value send to backend when the user summit the form